{"name": "test_change_user_profile_info", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=130.0.6723.69)\nStacktrace:\n0   chromedriver                        0x0000000104f6f634 cxxbridge1$str$ptr + 3645404\n1   chromedriver                        0x0000000104f67e94 cxxbridge1$str$ptr + 3614780\n2   chromedriver                        0x00000001049d4104 cxxbridge1$string$len + 88416\n3   chromedriver                        0x00000001049afb70 core::str::slice_error_fail::h1cab30ac4b13c655 + 3792\n4   chromedriver                        0x0000000104a3c66c cxxbridge1$string$len + 515784\n5   chromedriver                        0x0000000104a4f638 cxxbridge1$string$len + 593556\n6   chromedriver                        0x0000000104a0af54 cxxbridge1$string$len + 313264\n7   chromedriver                        0x0000000104a0bba4 cxxbridge1$string$len + 316416\n8   chromedriver                        0x0000000104f3a1d4 cxxbridge1$str$ptr + 3427196\n9   chromedriver                        0x0000000104f3d518 cxxbridge1$str$ptr + 3440320\n10  chromedriver                        0x0000000104f215f8 cxxbridge1$str$ptr + 3325856\n11  chromedriver                        0x0000000104f3dddc cxxbridge1$str$ptr + 3442564\n12  chromedriver                        0x0000000104f1287c cxxbridge1$str$ptr + 3265060\n13  chromedriver                        0x0000000104f58884 cxxbridge1$str$ptr + 3551788\n14  chromedriver                        0x0000000104f58a00 cxxbridge1$str$ptr + 3552168\n15  chromedriver                        0x0000000104f67b2c cxxbridge1$str$ptr + 3613908\n16  libsystem_pthread.dylib             0x0000000192c132e4 _pthread_start + 136\n17  libsystem_pthread.dylib             0x0000000192c0e0fc thread_start + 8", "trace": "self = <page.profile_account_page.ProfilePage object at 0x102054a00>, locator = ('xpath', \"//a[contains(@href, '/profile')]\"), timeout = 10\n\n    @allure.step(\"Ожидание видимости элемента: {locator}\")\n    def wait_for_element(self, locator, timeout=10):\n        try:\n>           element = WebDriverWait(self.driver, timeout).until(\n                expected_conditions.visibility_of_element_located(locator)\n            )\n\npage/base_page.py:17: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nvenv/lib/python3.9/site-packages/selenium/webdriver/support/wait.py:96: in until\n    value = method(self._driver)\nvenv/lib/python3.9/site-packages/selenium/webdriver/support/expected_conditions.py:156: in _predicate\n    return _element_if_visible(driver.find_element(*locator))\nvenv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py:748: in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\nvenv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py:354: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x102022f40>\nresponse = {'status': 404, 'value': '{\"value\":{\"error\":\"no such window\",\"message\":\"no such window: target window already closed\\\\...000192c132e4 _pthread_start + 136\\\\n17  libsystem_pthread.dylib             0x0000000192c0e0fc thread_start + 8\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nE       from unknown error: web view not found\nE         (Session info: chrome=130.0.6723.69)\nE       Stacktrace:\nE       0   chromedriver                        0x0000000104f6f634 cxxbridge1$str$ptr + 3645404\nE       1   chromedriver                        0x0000000104f67e94 cxxbridge1$str$ptr + 3614780\nE       2   chromedriver                        0x00000001049d4104 cxxbridge1$string$len + 88416\nE       3   chromedriver                        0x00000001049afb70 core::str::slice_error_fail::h1cab30ac4b13c655 + 3792\nE       4   chromedriver                        0x0000000104a3c66c cxxbridge1$string$len + 515784\nE       5   chromedriver                        0x0000000104a4f638 cxxbridge1$string$len + 593556\nE       6   chromedriver                        0x0000000104a0af54 cxxbridge1$string$len + 313264\nE       7   chromedriver                        0x0000000104a0bba4 cxxbridge1$string$len + 316416\nE       8   chromedriver                        0x0000000104f3a1d4 cxxbridge1$str$ptr + 3427196\nE       9   chromedriver                        0x0000000104f3d518 cxxbridge1$str$ptr + 3440320\nE       10  chromedriver                        0x0000000104f215f8 cxxbridge1$str$ptr + 3325856\nE       11  chromedriver                        0x0000000104f3dddc cxxbridge1$str$ptr + 3442564\nE       12  chromedriver                        0x0000000104f1287c cxxbridge1$str$ptr + 3265060\nE       13  chromedriver                        0x0000000104f58884 cxxbridge1$str$ptr + 3551788\nE       14  chromedriver                        0x0000000104f58a00 cxxbridge1$str$ptr + 3552168\nE       15  chromedriver                        0x0000000104f67b2c cxxbridge1$str$ptr + 3613908\nE       16  libsystem_pthread.dylib             0x0000000192c132e4 _pthread_start + 136\nE       17  libsystem_pthread.dylib             0x0000000192c0e0fc thread_start + 8\n\nvenv/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py:229: NoSuchWindowException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <test_personal_account.TestPersonalAccount object at 0x101db1a90>\nopen_browser = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"eff682654ccfc77756481b61c430054e\")>\n\n    @allure.step(\"Изменение информации в профиле пользователя\")\n    def test_change_user_profile_info(self, open_browser):\n        profile_page = ProfilePage(open_browser)\n        profile_page.open_user_orders()\n>       profile_page.go_to_profile_tab()\n\ntests/test_personal_account.py:42: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npage/profile_account_page.py:23: in go_to_profile_tab\n    self.action_click(self.locators.PROFILE_TAB)\npage/base_page.py:44: in action_click\n    element = self.wait_for_element(locator)\npage/base_page.py:23: in wait_for_element\n    self.driver.save_screenshot('wait_for_element_exception.png')\nvenv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py:829: in save_screenshot\n    return self.get_screenshot_as_file(filename)\nvenv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py:805: in get_screenshot_as_file\n    png = self.get_screenshot_as_png()\nvenv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py:839: in get_screenshot_as_png\n    return b64decode(self.get_screenshot_as_base64().encode(\"ascii\"))\nvenv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py:850: in get_screenshot_as_base64\n    return self.execute(Command.SCREENSHOT)[\"value\"]\nvenv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py:354: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x102022f40>\nresponse = {'status': 404, 'value': '{\"value\":{\"error\":\"no such window\",\"message\":\"no such window: target window already closed\\\\...000192c132e4 _pthread_start + 136\\\\n17  libsystem_pthread.dylib             0x0000000192c0e0fc thread_start + 8\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nE       from unknown error: web view not found\nE         (Session info: chrome=130.0.6723.69)\nE       Stacktrace:\nE       0   chromedriver                        0x0000000104f6f634 cxxbridge1$str$ptr + 3645404\nE       1   chromedriver                        0x0000000104f67e94 cxxbridge1$str$ptr + 3614780\nE       2   chromedriver                        0x00000001049d4104 cxxbridge1$string$len + 88416\nE       3   chromedriver                        0x00000001049afb70 core::str::slice_error_fail::h1cab30ac4b13c655 + 3792\nE       4   chromedriver                        0x0000000104a3c66c cxxbridge1$string$len + 515784\nE       5   chromedriver                        0x0000000104a4f638 cxxbridge1$string$len + 593556\nE       6   chromedriver                        0x0000000104a0af54 cxxbridge1$string$len + 313264\nE       7   chromedriver                        0x0000000104a0bba4 cxxbridge1$string$len + 316416\nE       8   chromedriver                        0x0000000104f3a1d4 cxxbridge1$str$ptr + 3427196\nE       9   chromedriver                        0x0000000104f3d518 cxxbridge1$str$ptr + 3440320\nE       10  chromedriver                        0x0000000104f215f8 cxxbridge1$str$ptr + 3325856\nE       11  chromedriver                        0x0000000104f3dddc cxxbridge1$str$ptr + 3442564\nE       12  chromedriver                        0x0000000104f1287c cxxbridge1$str$ptr + 3265060\nE       13  chromedriver                        0x0000000104f58884 cxxbridge1$str$ptr + 3551788\nE       14  chromedriver                        0x0000000104f58a00 cxxbridge1$str$ptr + 3552168\nE       15  chromedriver                        0x0000000104f67b2c cxxbridge1$str$ptr + 3613908\nE       16  libsystem_pthread.dylib             0x0000000192c132e4 _pthread_start + 136\nE       17  libsystem_pthread.dylib             0x0000000192c0e0fc thread_start + 8\n\nvenv/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py:229: NoSuchWindowException"}, "steps": [{"name": "Изменение информации в профиле пользователя", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=130.0.6723.69)\nStacktrace:\n0   chromedriver                        0x0000000104f6f634 cxxbridge1$str$ptr + 3645404\n1   chromedriver                        0x0000000104f67e94 cxxbridge1$str$ptr + 3614780\n2   chromedriver                        0x00000001049d4104 cxxbridge1$string$len + 88416\n3   chromedriver                        0x00000001049afb70 core::str::slice_error_fail::h1cab30ac4b13c655 + 3792\n4   chromedriver                        0x0000000104a3c66c cxxbridge1$string$len + 515784\n5   chromedriver                        0x0000000104a4f638 cxxbridge1$string$len + 593556\n6   chromedriver                        0x0000000104a0af54 cxxbridge1$string$len + 313264\n7   chromedriver                        0x0000000104a0bba4 cxxbridge1$string$len + 316416\n8   chromedriver                        0x0000000104f3a1d4 cxxbridge1$str$ptr + 3427196\n9   chromedriver                        0x0000000104f3d518 cxxbridge1$str$ptr + 3440320\n10  chromedriver                        0x0000000104f215f8 cxxbridge1$str$ptr + 3325856\n11  chromedriver                        0x0000000104f3dddc cxxbridge1$str$ptr + 3442564\n12  chromedriver                        0x0000000104f1287c cxxbridge1$str$ptr + 3265060\n13  chromedriver                        0x0000000104f58884 cxxbridge1$str$ptr + 3551788\n14  chromedriver                        0x0000000104f58a00 cxxbridge1$str$ptr + 3552168\n15  chromedriver                        0x0000000104f67b2c cxxbridge1$str$ptr + 3613908\n16  libsystem_pthread.dylib             0x0000000192c132e4 _pthread_start + 136\n17  libsystem_pthread.dylib             0x0000000192c0e0fc thread_start + 8\n\n", "trace": "  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/tests/test_personal_account.py\", line 42, in test_change_user_profile_info\n    profile_page.go_to_profile_tab()\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/page/profile_account_page.py\", line 23, in go_to_profile_tab\n    self.action_click(self.locators.PROFILE_TAB)\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/page/base_page.py\", line 44, in action_click\n    element = self.wait_for_element(locator)\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/page/base_page.py\", line 23, in wait_for_element\n    self.driver.save_screenshot('wait_for_element_exception.png')\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 829, in save_screenshot\n    return self.get_screenshot_as_file(filename)\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 805, in get_screenshot_as_file\n    png = self.get_screenshot_as_png()\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 839, in get_screenshot_as_png\n    return b64decode(self.get_screenshot_as_base64().encode(\"ascii\"))\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 850, in get_screenshot_as_base64\n    return self.execute(Command.SCREENSHOT)[\"value\"]\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Переход на страницу заказов", "status": "passed", "steps": [{"name": "Переход по URL: 'https://stellarburgers.nomoreparties.site/account/profile'", "status": "passed", "steps": [{"name": "Ожидание видимости элемента: ('xpath', \"//div[contains(@class, 'profile__order')]\")", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', \"//div[contains(@class, 'profile__order')]\")"}, {"name": "timeout", "value": "10"}], "start": 1729784799665, "stop": 1729784810011}], "parameters": [{"name": "url", "value": "'https://stellarburgers.nomoreparties.site/account/profile'"}, {"name": "expected_element", "value": "('xpath', \"//div[contains(@class, 'profile__order')]\")"}], "start": 1729784799299, "stop": 1729784810011}], "start": 1729784799299, "stop": 1729784810011}, {"name": "Переход на профиль", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=130.0.6723.69)\nStacktrace:\n0   chromedriver                        0x0000000104f6f634 cxxbridge1$str$ptr + 3645404\n1   chromedriver                        0x0000000104f67e94 cxxbridge1$str$ptr + 3614780\n2   chromedriver                        0x00000001049d4104 cxxbridge1$string$len + 88416\n3   chromedriver                        0x00000001049afb70 core::str::slice_error_fail::h1cab30ac4b13c655 + 3792\n4   chromedriver                        0x0000000104a3c66c cxxbridge1$string$len + 515784\n5   chromedriver                        0x0000000104a4f638 cxxbridge1$string$len + 593556\n6   chromedriver                        0x0000000104a0af54 cxxbridge1$string$len + 313264\n7   chromedriver                        0x0000000104a0bba4 cxxbridge1$string$len + 316416\n8   chromedriver                        0x0000000104f3a1d4 cxxbridge1$str$ptr + 3427196\n9   chromedriver                        0x0000000104f3d518 cxxbridge1$str$ptr + 3440320\n10  chromedriver                        0x0000000104f215f8 cxxbridge1$str$ptr + 3325856\n11  chromedriver                        0x0000000104f3dddc cxxbridge1$str$ptr + 3442564\n12  chromedriver                        0x0000000104f1287c cxxbridge1$str$ptr + 3265060\n13  chromedriver                        0x0000000104f58884 cxxbridge1$str$ptr + 3551788\n14  chromedriver                        0x0000000104f58a00 cxxbridge1$str$ptr + 3552168\n15  chromedriver                        0x0000000104f67b2c cxxbridge1$str$ptr + 3613908\n16  libsystem_pthread.dylib             0x0000000192c132e4 _pthread_start + 136\n17  libsystem_pthread.dylib             0x0000000192c0e0fc thread_start + 8\n\n", "trace": "  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/page/profile_account_page.py\", line 23, in go_to_profile_tab\n    self.action_click(self.locators.PROFILE_TAB)\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/page/base_page.py\", line 44, in action_click\n    element = self.wait_for_element(locator)\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/page/base_page.py\", line 23, in wait_for_element\n    self.driver.save_screenshot('wait_for_element_exception.png')\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 829, in save_screenshot\n    return self.get_screenshot_as_file(filename)\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 805, in get_screenshot_as_file\n    png = self.get_screenshot_as_png()\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 839, in get_screenshot_as_png\n    return b64decode(self.get_screenshot_as_base64().encode(\"ascii\"))\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 850, in get_screenshot_as_base64\n    return self.execute(Command.SCREENSHOT)[\"value\"]\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Клик по элементу: ('xpath', \"//a[contains(@href, '/profile')]\")", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=130.0.6723.69)\nStacktrace:\n0   chromedriver                        0x0000000104f6f634 cxxbridge1$str$ptr + 3645404\n1   chromedriver                        0x0000000104f67e94 cxxbridge1$str$ptr + 3614780\n2   chromedriver                        0x00000001049d4104 cxxbridge1$string$len + 88416\n3   chromedriver                        0x00000001049afb70 core::str::slice_error_fail::h1cab30ac4b13c655 + 3792\n4   chromedriver                        0x0000000104a3c66c cxxbridge1$string$len + 515784\n5   chromedriver                        0x0000000104a4f638 cxxbridge1$string$len + 593556\n6   chromedriver                        0x0000000104a0af54 cxxbridge1$string$len + 313264\n7   chromedriver                        0x0000000104a0bba4 cxxbridge1$string$len + 316416\n8   chromedriver                        0x0000000104f3a1d4 cxxbridge1$str$ptr + 3427196\n9   chromedriver                        0x0000000104f3d518 cxxbridge1$str$ptr + 3440320\n10  chromedriver                        0x0000000104f215f8 cxxbridge1$str$ptr + 3325856\n11  chromedriver                        0x0000000104f3dddc cxxbridge1$str$ptr + 3442564\n12  chromedriver                        0x0000000104f1287c cxxbridge1$str$ptr + 3265060\n13  chromedriver                        0x0000000104f58884 cxxbridge1$str$ptr + 3551788\n14  chromedriver                        0x0000000104f58a00 cxxbridge1$str$ptr + 3552168\n15  chromedriver                        0x0000000104f67b2c cxxbridge1$str$ptr + 3613908\n16  libsystem_pthread.dylib             0x0000000192c132e4 _pthread_start + 136\n17  libsystem_pthread.dylib             0x0000000192c0e0fc thread_start + 8\n\n", "trace": "  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/page/base_page.py\", line 44, in action_click\n    element = self.wait_for_element(locator)\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/page/base_page.py\", line 23, in wait_for_element\n    self.driver.save_screenshot('wait_for_element_exception.png')\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 829, in save_screenshot\n    return self.get_screenshot_as_file(filename)\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 805, in get_screenshot_as_file\n    png = self.get_screenshot_as_png()\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 839, in get_screenshot_as_png\n    return b64decode(self.get_screenshot_as_base64().encode(\"ascii\"))\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 850, in get_screenshot_as_base64\n    return self.execute(Command.SCREENSHOT)[\"value\"]\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Ожидание видимости элемента: ('xpath', \"//a[contains(@href, '/profile')]\")", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=130.0.6723.69)\nStacktrace:\n0   chromedriver                        0x0000000104f6f634 cxxbridge1$str$ptr + 3645404\n1   chromedriver                        0x0000000104f67e94 cxxbridge1$str$ptr + 3614780\n2   chromedriver                        0x00000001049d4104 cxxbridge1$string$len + 88416\n3   chromedriver                        0x00000001049afb70 core::str::slice_error_fail::h1cab30ac4b13c655 + 3792\n4   chromedriver                        0x0000000104a3c66c cxxbridge1$string$len + 515784\n5   chromedriver                        0x0000000104a4f638 cxxbridge1$string$len + 593556\n6   chromedriver                        0x0000000104a0af54 cxxbridge1$string$len + 313264\n7   chromedriver                        0x0000000104a0bba4 cxxbridge1$string$len + 316416\n8   chromedriver                        0x0000000104f3a1d4 cxxbridge1$str$ptr + 3427196\n9   chromedriver                        0x0000000104f3d518 cxxbridge1$str$ptr + 3440320\n10  chromedriver                        0x0000000104f215f8 cxxbridge1$str$ptr + 3325856\n11  chromedriver                        0x0000000104f3dddc cxxbridge1$str$ptr + 3442564\n12  chromedriver                        0x0000000104f1287c cxxbridge1$str$ptr + 3265060\n13  chromedriver                        0x0000000104f58884 cxxbridge1$str$ptr + 3551788\n14  chromedriver                        0x0000000104f58a00 cxxbridge1$str$ptr + 3552168\n15  chromedriver                        0x0000000104f67b2c cxxbridge1$str$ptr + 3613908\n16  libsystem_pthread.dylib             0x0000000192c132e4 _pthread_start + 136\n17  libsystem_pthread.dylib             0x0000000192c0e0fc thread_start + 8\n\n", "trace": "  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/page/base_page.py\", line 23, in wait_for_element\n    self.driver.save_screenshot('wait_for_element_exception.png')\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 829, in save_screenshot\n    return self.get_screenshot_as_file(filename)\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 805, in get_screenshot_as_file\n    png = self.get_screenshot_as_png()\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 839, in get_screenshot_as_png\n    return b64decode(self.get_screenshot_as_base64().encode(\"ascii\"))\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 850, in get_screenshot_as_base64\n    return self.execute(Command.SCREENSHOT)[\"value\"]\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "parameters": [{"name": "locator", "value": "('xpath', \"//a[contains(@href, '/profile')]\")"}, {"name": "timeout", "value": "10"}], "start": 1729784810012, "stop": 1729784818745}], "parameters": [{"name": "locator", "value": "('xpath', \"//a[contains(@href, '/profile')]\")"}, {"name": "expected_element", "value": "None"}], "start": 1729784810012, "stop": 1729784818751}], "start": 1729784810011, "stop": 1729784818751}], "parameters": [{"name": "open_browser", "value": "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"eff682654ccfc77756481b61c430054e\")>"}], "start": 1729784799299, "stop": 1729784818751}], "attachments": [{"name": "stdout", "source": "23c39c68-9983-4fc0-8dc3-929eddce1ade-attachment.txt", "type": "text/plain"}], "start": 1729784799299, "stop": 1729784818752, "uuid": "3f257e9b-290d-4e16-833a-93c7409ad005", "historyId": "b4c613f7e5387e67a7f956014dae1c53", "testCaseId": "b4c613f7e5387e67a7f956014dae1c53", "fullName": "tests.test_personal_account.TestPersonalAccount#test_change_user_profile_info", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('login')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_personal_account"}, {"name": "subSuite", "value": "TestPersonalAccount"}, {"name": "host", "value": "MacBook-Air-Artem.local"}, {"name": "thread", "value": "71652-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_personal_account"}]}